src/game/client/draw/w3dmodeldraw.cpp

```diff
@@ -144,7 +144,7 @@ void ModelConditionInfo::Validate_Stuff(RenderObjClass *robj, float scale, std::
     Validate_Weapon_Barrel_Info();
 }
 
-bool Test_Animation_Flag(int flags, char flag)
+bool Test_Animation_Flag(int flags, AnimationStateFlag flag)

```

Comment:
This is a more explicit. No need for it to be a `char` when we pass `AnimationStateFlag` to it.